infrastructure/proto/log_replication_cluster_info.proto

```diff
@@ -31,6 +31,5 @@ enum ClusterRole {
   INVALID = 0;
   ACTIVE = 1;
   STANDBY = 2;
-  NONE = 3;
```

Comment:
This NONE is still used in NSX code

Reply:
Ohh, I was wondering if we can merge INVALID and NONE as they might be serving a similar purpose. Can we do it or we need both? I found INVALID was used in many places within our codebase, and hence tried to replace it with NONE.

Reply:
I think right now we need both, as they are corresponding roles of NSX side.

Reply:
One more question about other enums - Does your change affect version compatibility?

Let's assume we have an old runtime has the INVALID = 0, and a new server that INVALID has been removed.
Now the new server has READY = 0, and receives a runtime request with INVALID. Does the new server know the request is INVALID/UNRECOGNIZED, or it will think the request is READY=0?

Reply:
You are right. It will think the request is `READY=0`. The numbers assigned to the enum are given priority while serializing and de-serializing. 

I thought these changes are fine for now because from the next major release onwards, we will be supporting backward compatibility and we will have new compatibility vectors. Until then, we are good to make these breaking changes. It will work fine as long as the client and server are in the same version.

Also, in proto3, the First enum value must be 0, So we can't really keep the same numbers for the remaining values and just remove the `INVALID = 0` enum value.